### PR TITLE
[MNT]: 2019.1.0rc2 prep

### PR DESCRIPTION
**Pull request by asreimer**
Housekeeping. Bumping version numbers, updating valid resen-cores, etc.

### CODE REVIEW

**Comment by asreimer** [comment]
Waiting for #31 then will review codebase and head towards release.

**Comment by pmreyes2** [comment]
I'll test this. 
```
...adding core...
...adding mounts...
Bucket created successfully!
Pulling image: 2019.1.0rc2
   This may take some time...
[==================================================>] 100.00 %, 2.667/2.67GB Elapsed time: 0:02:10
Done!
...starting jupyterlab...
ERROR: Failed to start jupyter server!
[resen] >>>
```

The docker container is running, and the patched file:
/home/jovyan/envs/py36/lib/python3.6/site-packages/notebook/notebookapp.py looks good

and I have no problems starting a jupyter lab from a terminal in the container:
`docker exec -it --user jovyan resen_b0 bash`
Only thing strange is that it brings me to / instead of /home/jovyan

- Besides kernels py27 and py36, kernel python3 still shows as an option in jupyter, but it is essentially py36.

Tested on mac and on ubuntu server:
As a test, I added both rc1 and rc2 to the resen tool, so:
`Available versions: 2019.1.0rc1, 2019.1.0rc2`
Created buckets with both versions. 2019.1.0rc1 starts/stops(fixing notebookapp.py manually) jupyter fine, but starting a bucket with image 2019.1.0rc2 fails to start jupyter:
```
...adding core...
...adding mounts...
Bucket created successfully!
...starting jupyterlab...
ERROR: Failed to start jupyter server!
```

**Comment by ljlamarche** [comment]
I seem to be having the same issue as @pmreyes2 .
```
>>> Start bucket and jupyterlab? (y/n): y
...adding core...
...adding mounts...
Bucket created successfully!
...starting jupyterlab...
ERROR: Failed to start jupyter server!
```
I'll start digging into resen-core 2019.1.0rc2 and see if I can identify an issue.

File: resen/Resen.py [hunk 1]
```diff
@@ -750,7 +750,7 @@ def update_bar(sum_total,accumulated,t0,current_time, scale=0.5):
             nchars = int(percentage*scale)
             bar = "\r["+nchars*"="+">"+(int(100*scale)-nchars)*" "+"]"
             time_info = "Elapsed time: %s"%truncate_secs(current_time - t0)
-            print(bar+" %5.2f %%, %5.3f/%4.2fGB %s"%(percentage,
+            print(bar+" %6.2f %%, %5.3f/%4.2fGB %s"%(percentage,
```
Comment (pmreyes2) [inline resen/Resen.py:753]: This is a very small typo fixed: %6.2f to hold a 100.00 number

**Comment by pmreyes2** [comment]
Everything working great. Tested on mac and linux.

**Pull request merged**